### ISSUE TITLE
Remove Checkout on Single Product button from Grouped Products

### ISSUE THREAD

**Issue by johndcoy-zz** [woocommerce/woocommerce-gateway-paypal-express-checkout#478]
The Grouped product type is not currently supported so it would be helpful to hide the button for this product type or enhance the plugin to support grouped simple products.

This CSS can be used for now:

```
.product-type-grouped .wcppec-checkout-buttons {
    display: none;
}
```

**Comment by davidfguk** [comment]
The above worked perfect (CSS hide on Grouped) white it doesnt work...

Either a checkbox to disable on Group Products within the in settings  of PayPal Checkout
or make it work with Grouped Products out of the box.

**Comment by jessepearson** [comment]
Moving this from an Enhancement to a Bug due to the button shouldn't show if the product isn't compatible. 
12540933-hc

**Comment by mattallan** [comment]
Fixed with #663
